article1.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Article 1</title>
+  </head>
+  <body>
+    <nav class="socialNav">
+      <ul>
+        <li><a href="#">Facebook</a></li>
```

Comment:
I would omit the `<ul> <li>` structure for links - it only needs a `nav` element as the parent, and `a`'s as the children.